src/test/kotlin/no/skatteetaten/aurora/boober/feature/AuroraAzureAppSubPartTest.kt

```diff
@@ -231,4 +231,16 @@ class AuroraAzureAppSubPartTest : AbstractMultiFeatureTest() {
         assertThat(alternativeRoute).auroraResourceCreatedByTarget(AzureFeature::class.java)
             .auroraResourceMatchesFile("aurora-azure-route-timeout.json")
     }
+
```

Comment:
Burde ha en test for azure: true siden du implisitt enabler at det kan gjøres. Jeg kjenner ikke til hva du ønsker skal skje i det tilfellet, om det skal validere feil eller gå ok uten at noe skjer, eller at det skal generere en warning. 

Kanskje man burde vurdere å ha azureDefaults hvis det er hensiktsmessig slik at man kan definere azureDefaults i about filer og kun azure: true i app filer f.eks

Reply:
Poenget er egentlig det motsatte; noen vil definere azure-definisjonen i base-filen og kunne sette azure: false i underfiler for for eksempel systemtester. Dersom man setter "azure: true" uten ytterligere konfig, vil det ikke skje noe med det (de ulike features under Azure-taggen håndteres i https://github.com/Skatteetaten/boober/pull/567/files/c09c40ee75d4618ab002bd51f27db8adb55d8171#diff-9ce8fd0d22788dbdedd1d0f42c2649a34d11aba5ae3f5bea86ced3c96c55bcf8R41 ).

Reply:
Burde man vurdere å heller ha `azure/enabled` istedenfor den simplified configen. Det virker litt rart at man kan si azure: true uten at noe skjer etter min mening

Reply:
Jeg har laget en test som viser dette.

Reply:
azure/enabled prøvde jeg ut først. Det vil da medføre at alle som allerede har byttet over må legge til en ny linje i deres config - dette vil jeg veldig gjerne unngå. 

Alternativt kan jeg bruke samme logikk der; den som har en azure-tagg men ingen azure/enabled-tagg har en "skjult" `azure/enabled: true`. Det vil da medføre noe tilsvarende kode ( `isAzureSpecificallyDisabled() = spec.getOrNull<Boolean>("azure/enabled") == false` )

Reply:
Hvis det er mulig å gjøre det du foreslår i siste avsnitt så er `azure/enabled` å foretrekke.

Reply:
Det ble noe herk å ha et "usynlig felt" - det var både enklere å lese og lettere å forstå med et simplified-felt.